compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Version.hs

```diff
@@ -143,6 +143,15 @@ featureTypeInterning = Feature
     , featureCppFlag = Nothing
     }
 
+featureExceptions :: Feature
+featureExceptions = Feature
+    { featureName = "DAML Exceptions"
+    , featureMinVersion = versionDev
+        -- TODO (#8020): Update LF version number when we stabilize exceptions.
+        -- https://github.com/digital-asset/daml/issues/8020
```

Comment:
I think this is a place where we should definitely leave a link to the issue for freezing the next DAML-LF version. (IIRC it is #7139.) We might consider exceptions done and #8020 closed before we make that freeze.

Reply:
Added the URL!